waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -821,8 +832,21 @@ proc mountRlnRelayDynamic*(node: WakuNode,
 
   node.wakuRlnRelay = rlnPeer
 
+proc readPersistentRlnCredentials*(path: string) : RlnMembershipCredentials {.raises: [Defect, OSError, IOError, Exception].} =
+  info "Rln credentials exist in file"
+  # With regards to printing the keys, it is purely for debugging purposes so that the user becomes explicitly aware of the current keys in use when nwaku is started.
+  # Note that this is only until the RLN contract being used is the one deployed on Goerli testnet.
+  # These prints need to omitted once RLN contract is deployed on Ethereum mainnet and using valuable funds for staking.
+      
+  let entireRlnCredentialsFile = readFile(path)
+
+  let jsonObject = parseJson(entireRlnCredentialsFile)
+  let deserializedRlnCredentials = to(jsonObject, RlnMembershipCredentials)
+  
+  debug "Deserialized Rln credentials", rlnCredentials=deserializedRlnCredentials
```

Comment:
I would prefer that we do not dump the content of the credentials through the stdout

Reply:
Refer to https://github.com/status-im/nwaku/pull/1037#issuecomment-1191167672